src/api.cpp

```diff
@@ -2,20 +2,26 @@
 #include <fstream>
 #include <map>
 #include <string>
+#include <vector>
 
 #include <libreborn/libreborn.h>
+#include <symbols/minecraft.h>
 #include <mods/misc/misc.h>
+#include <mods/chat/chat.h>
 
 #include "api.h"
 #include "base64.h"
 #include "extra.h"
 #include "helpers.h"
 
+std::vector<std::string> chat_log = {};
```

Comment:
Could this be moved closer to where it's actually used?

Reply:
I put it there because other stuff was being defined in that area but sure, I'll do that.

Reply:
Nevermind! I forgot that I must define stuff before I define the function, and putting it between the `handle_pollChatPosts` and previous functions looks weird.